server/channels/store/storetest/channel_store.go

```diff
@@ -371,13 +371,19 @@ func testChannelStoreUpdate(t *testing.T, rctx request.CTX, ss store.Store) {
 }
 
 func testGetChannelUnread(t *testing.T, rctx request.CTX, ss store.Store) {
-	teamId1 := model.NewId()
-	teamId2 := model.NewId()
+	teamID1 := model.NewId()
+	teamID2 := model.NewId()
 
 	uid := model.NewId()
+<<<<<<< HEAD
 	m1 := &model.TeamMember{TeamId: teamId1, UserId: uid}
 	m2 := &model.TeamMember{TeamId: teamId2, UserId: uid}
 	_, nErr := ss.Team().SaveMember(rctx, m1, -1)
+=======
+	m1 := &model.TeamMember{TeamId: teamID1, UserId: uid}
+	m2 := &model.TeamMember{TeamId: teamID2, UserId: uid}
+	_, nErr := ss.Team().SaveMember(m1, -1)
+>>>>>>> fixed var-naming issues identified by go.lint-revive in 13 files
```

Comment:
There is a merge conflict to resolve here.

Reply:
My bad, fixed.